blockly-core/core/ui/block_space/pan_drag_handler.js

```diff
@@ -227,6 +283,22 @@ Blockly.PanDragHandler.prototype.beginDragScroll_ = function (e) {
 };
 
 /**
+ * Given desired new scrollX and scrollY positions
+ * @param {number} newScrollX new target pan-right (+) offset
+ * @param {number} newScrollY new target pan-down (+) offset
+ */
+Blockly.PanDragHandler.prototype.scrollTo = function (newScrollX, newScrollY) {
```

Comment:
After this changeset this method seems [envious](http://c2.com/cgi/wiki?FeatureEnvySmell) of `BlockSpace`, moving there